### DISCUSSION TITLE
How do I change the location of temporary files ?

### DISCUSSION THREAD

**Discussion by l3gi0n0fh3ll** [texstudio-org/texstudio#1713] [General]
When viewing quick previews of equations through **Preview Selection/Parentheses** or **tooltips**, temporary files are created at: C:\Users\user-name\AppData\Local\Temp ... Is there a way to change this location to somewhere else ?

After running Help --> Check Latex Installation, I get these two lines:
 
TEMP=C:\Users\<user-name>\AppData\Local\Temp
TMP=C:\Users\<user-name>\AppData\Local\Temp

I am guessing I have to change the TEMP and TMP values to something else so that the temp files are stored where I want them to be...How do I achieve this ?

**Comment by sunderme** [comment]
https://doc.qt.io/qt-5/qdir.html#tempPath

**Comment by sunderme** [comment]
you can google how to set variables on windows

**Comment by l3gi0n0fh3ll** [comment]
But wouldn't this change the temp dir for all programs ? I just want to change the temp dir for TexStudio...

**Reply by sunderme** [reply]
I don't work on windows, so I would have to google as well ...
